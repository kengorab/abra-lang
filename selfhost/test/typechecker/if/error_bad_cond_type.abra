val a = if 123 { "123" } else 456