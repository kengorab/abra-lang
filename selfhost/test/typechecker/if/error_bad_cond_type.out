Error at %FILE_NAME%:1:12
Forbidden type for if-condition
  |  val a = if 123 { "123" } else 456
                ^
Conditions must either be Bool or Option types
but instead found Int
